app/services/control_rate_service.rb

```diff
@@ -58,7 +58,7 @@ def sum_cumulative_registrations
   def registration_counts
     return @registration_counts if defined? @registration_counts
     formatter = lambda { |v| quarterly_report? ? Period.quarter(v) : Period.month(v) }
-    result = region.registered_patients.with_hypertension.group_by_period(periods.begin.type, :recorded_at, {format: formatter}).count
+    result = region.assigned_patients.with_hypertension.group_by_period(periods.begin.type, :recorded_at, {format: formatter}).count
```

Comment:
Oh, we're scoping `with_hypertension` here, want to use `assigned_hypertension_patients` here directly?

Reply:
That was my first thought as well. Facility groups don't have that interface available, so I decided to keep the interface simple and just compose what we need here. Facility groups do feature an `assigned_patients` assocation through facilities quite nicely.